src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/resource/VideoLinkBookingController.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.resource
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.tags.Tag
+import jakarta.validation.Valid
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.VideoLinkBookingCreateRequest
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.CreateVideoLinkBookingService
+
+@Tag(name = "Video Link Booking")
+@RestController
+@RequestMapping(value = ["video-link-booking"], produces = [MediaType.APPLICATION_JSON_VALUE])
+class VideoLinkBookingController(val createVideoLinkBookingService: CreateVideoLinkBookingService) {
+
+  @Operation(summary = "Endpoint to support the creation of video link bookings")
+  @PostMapping(consumes = [MediaType.APPLICATION_JSON_VALUE])
+  @ResponseStatus(HttpStatus.CREATED)
+  @PreAuthorize("hasAnyRole('BOOK_A_VIDEO_LINK_ADMIN')")
```

Comment:
I should add I am not 100% sure on the roles needed/required.

Reply:
BOOK_A_VIDEO_LINK_ADMIN is as good as any. Call should be made with the client_creds token, with the ?username=real-user (so we can record who was actually doing it).